tests/providers/google/marketing_platform/operators/test_display_video_system.py

```diff
@@ -26,14 +28,22 @@
     "https://www.googleapis.com/auth/cloud-platform"
 ]
 
+GMP_PARTNER_ID = os.environ.get("GMP_PARTNER_ID", 123)
+ENTITY_TYPE = "LineItem"
+
 
 @pytest.mark.system("google.marketing_platform")
 @pytest.mark.credential_file(GMP_KEY)
 class DisplayVideoSystemTest(GoogleSystemTest):
 
+    @staticmethod
+    def erf_bucket(partner_id, entity_type):
+        return f"gdbm-{partner_id}/entity/{{{{ ds_nodash }}}}.*.{entity_type}.json"
```

Comment:
pylint will be sad due to no docstring, should we add type annotations as well?

Reply:
hurry is not a good approach... sorry. :(

Reply:
Yes, I will add docstring and types.